drizzlepac/haputils/catalog_utils.py

```diff
@@ -1874,20 +1873,13 @@ def identify_sources(self, **pars):
             # the segmentation image too
             good_rows = []
             good_segm_rows_by_label = []
-            bad_segm_rows_by_label = []
             updated_table = None
             for i, old_row in enumerate(total_measurements_table):
```

Comment:
There is nothing wrong with this, but there is probably a more succinct way to write this for loop now that there is no else statement.

Reply:
Only the minimum necessary changes were done here as it is not understood why any modification was needed here at all.